lib/bokken/uploaders/document.ex

```diff
@@ -4,8 +4,26 @@ defmodule Bokken.Uploaders.Document do
   """
   use Waffle.Definition
   use Waffle.Ecto.Definition
+  @extension_whitelist ~w(.jpg .jpeg .gif .png)
+  @max_file_size 50_000_000
```

Comment:
Same here. I imagine someone uploading a Python script, for example. Those are not too heavy, as they are just text.